docs/api/web-contents.md

```diff
@@ -1089,6 +1089,9 @@ with native view, which developers have very limited control of. With the
 the devtools in it, including `BrowserWindow`, `BrowserView` and `<webview>`
 tag.
 
+Note that closing the devtools does not destory the `devToolsWebContents`, it
+is caller's responsibility to close `devToolsWebContents`.
```

Comment:
@zcbenz shouldn't this be:
Note that closing the devtools does not destroy the `devToolsWebContents`, it 
is caller's responsibility to destroy `devToolsWebContents`.